test-site/README.md

```diff
@@ -0,0 +1,15 @@
+### Answers Core Test Site
+
+This is a barebones site for testing the answers-core
+
+#### Instructions
+
+- Clone the answers-core
+- Inside answers-core, run `npm install` and `npm run build`
+
+- Clone this repo into the same directory that answers-core is in
```

Comment:
Do we need to update these instructions?

Reply:
yeah, updated!